ui/src/app/views/profile-page/component.ts

```diff
@@ -55,6 +55,20 @@ export class ProfilePageComponent implements OnInit {
   @ViewChild(BugReportComponent)
   bugReportComponent: BugReportComponent;
 
+  static valid(val: string, len: number): boolean {
```

Comment:
nit: existing issue but I'd probably make the function name more specific, e.g. `validLength`